# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/xre/moz.build
# Commit: 6d7485c81e90
# Full Hash: 6d7485c81e90e2d2e70dffc5014bcaaf34b18a87
# Author: Robert Mader <robert.mader@posteo.de>
# Date: 2022-06-21 15:52:43
# Description:
#   Bug 1758473 - Move VA-API test into glxtest, r=stransky
#   
#   Some VA-API drivers are so broken that trying to use them
#   crashes Firefox. This is nothing entirely new for GPU drivers
#   and which is why we have `glxtest`.
# ==============================================================================

diff -r 54aff5bcc909 -r 6d7485c81e90 toolkit/xre/moz.build
--- a/toolkit/xre/moz.build	Tue Jun 21 08:19:37 2022 +0000
+++ b/toolkit/xre/moz.build	Tue Jun 21 08:45:27 2022 +0000
@@ -160,6 +160,14 @@
         "glxtest.cpp",
     ]
 
+if CONFIG["MOZ_WAYLAND"]:
+    LOCAL_INCLUDES += [
+        "/media/mozva",
+    ]
+    USE_LIBS += [
+        "mozva",
+    ]
+
 if CONFIG["MOZ_INSTRUMENT_EVENT_LOOP"]:
     UNIFIED_SOURCES += [
         "EventTracer.cpp",